src/plugins/vue-imgix/index.ts

```diff
@@ -1,17 +1,17 @@
-import _Vue from 'vue';
+import _App from './App.vue';
```

Comment:
What does `./App.vue` resolve to?

Reply:
Oh wow, 😬  I forgot to push up a branch that precedes this. I'm going to have to force push here to make sure this has all the app. context. 

Basically, it comes down to `vue-class-component`'s `Vue` export.

```vue
// App.vue
<script>
import { Options, Vue } from "vue-class-component";
import HelloWorld from "./components/HelloWorld.vue";

@Options({
  components: {
    HelloWorld,
  },
})
export default class App extends Vue {}
</script>
```

```typescript
// index.ts
import { createApp } from "vue";
import App from "./App.vue";

createApp(App).mount("#app");

```

Reply:
Some more info on this here https://github.com/vuejs/vue-class-component/issues/406

Reply:
Thanks for the extra info. Looks like maybe the path for this is wrong?

Reply:
> Thanks for the extra info. Looks like maybe the path for this is wrong?

...💯  🤦🏼